weave-js/src/components/PagePanelComponents/Home/Browse2/WeaveEditors.tsx

```diff
@@ -13,37 +13,21 @@ import {
   Typography,
 } from '@mui/material';
 import {GridColDef, useGridApiRef} from '@mui/x-data-grid-pro';
-import {usePanelContext} from '@wandb/weave/components/Panel2/PanelContext';
-import {useWeaveContext} from '@wandb/weave/context';
 import {
-  constFunction,
-  constNumber,
-  constString,
   isAssignableTo,
-  isConstNode,
   isTypedDict,
-  linearize,
   listObjectType,
   maybe,
-  Node,
-  NodeOrVoidNode,
-  opDict,
-  opLimit,
-  opMap,
-  opObjGetAttr,
-  opPick,
-  OutputNode,
+  ObjectType,
```

Comment:
The vast majority of the changes in this file are to mograte the WeaveEditor to use the newly expose interface as opposed to directly interfacing with the weave engine. Practically, this means no more `Node`s!